syntax = "proto3";

package Ydb.Discovery.V1;
option java_package = "com.yandex.ydb.discovery.v1";

import "ydb/public/api/protos/ydb_discovery.proto";

service DiscoveryService {
    rpc ListEndpoints(Ydb.Discovery.ListEndpointsRequest) returns (Ydb.Discovery.ListEndpointsResponse);
    rpc WhoAmI(Ydb.Discovery.WhoAmIRequest) returns (Ydb.Discovery.WhoAmIResponse);
    rpc NodeRegistration(Ydb.Discovery.NodeRegistrationRequest) returns (Ydb.Discovery.NodeRegistrationResponse);
}
